src/Model/WineManager.php

```diff
@@ -89,19 +92,23 @@ public function checkIfFavorite(int $id): array|false
         return $statement->fetch();
     }
 
-    public function selectPartner(): array
+    public function selectWinesFromPartner(int $id): array
     {
-        $query = 'SELECT id, lastname, firstname FROM partner ';
+        $id = $_SESSION['pro_id'];
```

Comment:
Why do you take a $id in parameter in your function if you override it with the SESSION ?